pkgs/by-name/za/zatackax/package.nix

```
@@ -0,0 +1,44 @@
+{
+  autoreconfHook,
+  fetchFromGitHub,
+  SDL,
+  SDL_image,
+  SDL_mixer,
+  SDL_ttf,
+  stdenv,
+  lib,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zatackax";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "simenheg";
+    repo = "zatackax";
+    rev = "v${version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
We can use `tag` in `fetchFromGitHub` now.